go-controller/pkg/node/controllers/egressip/egressip_test.go

```diff
@@ -510,7 +510,31 @@ var _ = table.XDescribeTable("EgressIP selectors",
 							}
 						}
 					}
-
+				}
+				return nil
+			})
+		})
+		ginkgo.By("verify expected egress IPs are assigned to interface")
```

Comment:
@martinkennelly :  I'm having a hard time running this test. What am I missing?
```
❯ go test -mod=vendor -test.v -race github.com/ovn-org/ovn-kubernetes/go-controller/pkg/node/controllers/egressip
=== RUN   TestAdder
Running Suite: EgressIP Controller Suite
========================================
Random Seed: 1695411921
Will run 0 of 9 specs

P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures nothing when EIPs dont select anything
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:618
------------------------------
P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures one EIP and one Pod
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:631
------------------------------
P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures one EIP and multiple pods
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:655
------------------------------
P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures one EIP and multiple namespaces and multiple pods
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:687
------------------------------
P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures one EIP and multiple namespaces and multiple pods
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:720
------------------------------
P [PENDING]
EgressIP selectors
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:351
  configures multiple EIPs and multiple namespaces and multiple pods
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:752
------------------------------
P [PENDING]
repair node
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:808
  should not fail when node is clean and nothing to apply
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:866
------------------------------
P [PENDING]
repair node
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:808
  should remove stale route
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:879
------------------------------
P [PENDING]
repair node
/home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:808
  should remove stale address
  /home/vagrant/dev/ovn-kubernetes.git/go-controller/pkg/node/controllers/egressip/egressip_test.go:902
------------------------------

Ran 0 of 9 Specs in 0.003 seconds
SUCCESS! -- 0 Passed | 0 Failed | 9 Pending | 0 Skipped
--- PASS: TestAdder (0.00s)
PASS
ok      github.com/ovn-org/ovn-kubernetes/go-controller/pkg/node/controllers/egressip   1.087s
```

Reply:
@flavio-fernandes Its skipped because of an issue creating the netns in the GH VM. 
I havent had time to look into why yet but when i test locally they pass.

Reply:
See the comment at the top of the tests.